lms/djangoapps/django_comment_client/base/views.py

```diff
@@ -243,6 +243,7 @@ def update_thread(request, course_id, thread_id):
 
     course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)
     thread = cc.Thread.find(thread_id)
+    thread_context = getattr(thread, "context", "course")
```

Comment:
Because this bug is subtle, I think you should put a comment in the code explaining that it is necessary to make the getattr call before changing any properties of the thread.

I did a quick search for other usages of getattr in django_comment_client, and they appear to be safe.